src/elements/content-sidebar/ActivitySidebarFilter.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * @flow
+ * @file Activity feed sidebar filter component
+ * @author Box
+ */
+
+import * as React from 'react';
+import { FormattedMessage } from 'react-intl';
+import DropdownMenu, { MenuToggle } from '../../components/dropdown-menu';
```

Comment:
Minor, but we typically try to alphabetize our imports in groups, first-party and third-party.